src/cljs/main/broadfcui/page/library/library_page.cljs

```diff
@@ -186,11 +194,15 @@
                                                :from 0
                                                :size 10}))))}
         :typeaheadDisplay (fn [result]
-                            ;; we intentionally do not encode the result value here, because it is already
-                            ;; encoded from the server.
-                            (.text (js/$ (str "<div>" (aget result "value") "</div>"))))
+                            (let [sugg-object (js->clj (aget result "value"))
+                                  suggestion (get sugg-object "suggestion")]
+                              (.text (js/$ (str "<div>" suggestion "</div>")))))
```

Comment:
1. jQuery, internally, does not do tag-stripping. It looks like it relies on `textContent` to do browser-native HTML-to-text conversion. (see: https://github.com/jquery/jquery/blob/2.2-stable/external/sizzle/dist/sizzle.js#L978)

2. I think you may be misreading what the jquery text line is doing?
```
(.text (js/$ (str "<div>" suggestion "</div>")))
```
should be equivalent to: 
```
$("<div>" + suggestion + "</div>").text();
```
right? That statement creates - but does not append to the DOM - a <div> element, and then extracts its text content and returns that text content as a string.

3. if you're uncomfortable with the `.text()` approach, we'll need to update the impl from #738 too.

In general, I'm *more* prone to use a third-party library, especially a mature one like jQuery, than write our own encoder; that's why I raised a caveat about a hand-written encoder right up front!